fairscale/optim/oss.py

```diff
@@ -434,14 +431,14 @@ def _broadcast_params(
             # Since all the parameters are already sorted per increasing size, we only need to consider the first ones.
             while i_bucketed < len(params) and offset + params[i_bucketed].numel() < buffer_size:
                 end = offset + params[i_bucketed].numel()
-                if global_rank == self_rank:
+                if global_rank == self.global_rank:
```

Comment:
this was just changed on the previous commit to show that the test caught this mistake, not changed overall

Reply:
Doesn't it make sense to have two tests? One that checks the correct usage and one that checks incorrect usage?